openedx/features/qverse_features/registration/validators.py

```diff
@@ -0,0 +1,32 @@
+"""
+Validation utils for qverse registration application.
+"""
+from django.core.exceptions import ValidationError
+
+
+def validate_file_extension(value):
+    """
+    Validates that file is a valid csv file by checking its extension.
+    """
+    if not value.name.endswith('.csv'):
+        raise ValidationError(u'Please upload a valid csv file.')
+
+
+def validate_current_level(value):
+    """
+    Validates the provided current level.
+    """
+    # to avoid circular import
+    from .models import MAX_LEVEL_CHOICES
+    if not 1 <= value <= MAX_LEVEL_CHOICES:
+        raise ValidationError(u'Please provide valid current level.')
+
+
+def validate_programme_choices(value):
+    """
+    Validates the provided programme choice.
+    """
+    # to avoid circular import
+    from .models import MAX_PROGRAMME_CHOICES
+    if not 1 <= value <= MAX_PROGRAMME_CHOICES:
```

Comment:
```suggestion
    if value not in range(1,MAX_PROGRAMME_CHOICES):
```

Reply:
both are two different ways of doing a same thing. Both are somehow equally readable too. Keeping as it is.



BTW the syntax of your suggestion would be
```
if value not in range(1, MAX_PROGRAMME_CHOICES+1):
```